plugins/labhub.py

```diff
@@ -239,7 +247,13 @@ def newcomer_difficulty_level(user, iss):
             2. A newcomer asks for assignment to an issue with difficulty higher
                than low.
             """
-            if self.TEAMS[self.GH_ORG_NAME + ' newcomers'].is_member(user):
+            if (self.TEAMS[self.GH_ORG_NAME + ' newcomers'].is_member(user) and
```

Comment:
Ah ok, then it needs to be renamed as `newcomer_difficulty_level` implies True means newcomer difficulty level issues are allowed.